### PR TITLE
[doc](catalog)paimon doc

### PR DESCRIPTION
**Pull request by zddr**
## Proposed changes

Issue Number: close #xxx

code pr: https://github.com/apache/doris/pull/21910

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/docs/lakehouse/multi-catalog/paimon.md [hunk 1]
```diff
@@ -30,31 +30,76 @@ under the License.
 <version since="dev">
 </version>
 
-## 使用限制
+## 使用须知
 
-1. 目前只支持简单字段类型。
-2. 目前仅支持 Hive Metastore 类型的 Catalog。所以使用方式和 Hive Catalog 基本一致。后续版本将支持其他类型的 Catalog。
+1. 将 core-site.xml，hdfs-site.xml 和 hive-site.xml  放到 FE 和 BE 的 conf 目录下。优先读取 conf 目录下的 hadoop 配置文件，再读取环境变量 `HADOOP_CONF_DIR` 的相关配置文件。
```
Comment (morningman) [inline docs/zh-CN/docs/lakehouse/multi-catalog/paimon.md:35]: 只有使用hms作为元数据服务，或者数据存储在hdfs上时，才需要这些配置文件吧？

File: docs/zh-CN/docs/lakehouse/multi-catalog/paimon.md [hunk 2]
```diff
@@ -30,31 +30,76 @@ under the License.
...
-    "dfs.ha.namenodes.HDFS1006531" = "nn2,nn1",
-    "dfs.namenode.rpc-address.HDFS1006531.nn2" = "172.16.65.115:4007",
-    "dfs.namenode.rpc-address.HDFS1006531.nn1" = "172.16.65.15:4007",
-    "dfs.nameservices" = "HDFS1006531",
-    "hadoop.username" = "hadoop",
-    "warehouse" = "hdfs://HDFS1006531/data/paimon",
-    "dfs.client.failover.proxy.provider.HDFS1006531" = "org.apache.hadoop.hdfs.server.namenode.ha.ConfiguredFailoverProxyProvider"
+    "warehouse" = "s3://paimon-1308700295.cos.ap-beijing.myqcloud.com/paimoncos",
+    "s3.endpoint"="cos.ap-beijing.myqcloud.com",
+    "s3.access_key"="XXX",
```
Comment (morningman) [inline docs/zh-CN/docs/lakehouse/multi-catalog/paimon.md:68]:
```suggestion
    "s3.access_key"="ak",
```

File: docs/zh-CN/docs/lakehouse/multi-catalog/paimon.md [hunk 3]
```diff
@@ -30,31 +30,76 @@ under the License.
...
+    "s3.secret_key"="YYY"
```
Comment (morningman) [inline docs/zh-CN/docs/lakehouse/multi-catalog/paimon.md:69]:
```suggestion
    "s3.secret_key"="sk"
```

File: docs/zh-CN/docs/lakehouse/multi-catalog/paimon.md [hunk 4]
```diff
@@ -30,31 +30,76 @@ under the License.
...
+```
+
+#### OSS
+
+```sql
+   CREATE CATALOG `paimon_oss` PROPERTIES (
+    "type" = "paimon",
+    "warehouse" = "oss://paimon-zd/paimonoss",
+    "oss.endpoint"="oss-cn-beijing.aliyuncs.com",
+    "oss.access_key"="XXX",
```
Comment (morningman) [inline docs/zh-CN/docs/lakehouse/multi-catalog/paimon.md:81]: ditto

**Review by morningman — COMMENTED** [review]
English doc

File: docs/zh-CN/docs/lakehouse/multi-catalog/paimon.md (hunk 1 above)
Comment (zddr) [reply docs/zh-CN/docs/lakehouse/multi-catalog/paimon.md:35]: 对，我换个位置

**Comment by zddr** [comment]
run buildall

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by morningman** [comment]
run buildall

File: docs/en/docs/lakehouse/multi-catalog/paimon.md [hunk 5]
```diff
@@ -30,31 +30,76 @@ under the License.
...
+    "hadoop.username"="hadoop"
+);
+
+```
+
+#### S3
 
 ```sql
-CREATE CATALOG `paimon` PROPERTIES (
+  CREATE CATALOG `paimon_s3` PROPERTIES (
```
Comment (kaijchen) [inline docs/en/docs/lakehouse/multi-catalog/paimon.md:64]:
```suggestion
CREATE CATALOG `paimon_s3` PROPERTIES (
```

File: docs/en/docs/lakehouse/multi-catalog/paimon.md [hunk 6]
```diff
@@ -30,31 +30,76 @@ under the License.
...
+    "s3.access_key"="ak",
+    "s3.secret_key"="sk"
 );
+
+```
+
+#### OSS
+
+```sql
+   CREATE CATALOG `paimon_oss` PROPERTIES (
```
Comment (kaijchen) [inline docs/en/docs/lakehouse/multi-catalog/paimon.md:77]:
```suggestion
CREATE CATALOG `paimon_oss` PROPERTIES (
```

File: docs/zh-CN/docs/lakehouse/multi-catalog/paimon.md [hunk 7]
```diff
@@ -30,31 +30,76 @@ under the License.
...
+    "hadoop.username"="hadoop"
+);
+
+```
+
+#### S3
 
 ```sql
-CREATE CATALOG `paimon` PROPERTIES (
+  CREATE CATALOG `paimon_s3` PROPERTIES (
```
Comment (kaijchen) [inline docs/zh-CN/docs/lakehouse/multi-catalog/paimon.md:64]:
```suggestion
CREATE CATALOG `paimon_s3` PROPERTIES (
```

File: docs/zh-CN/docs/lakehouse/multi-catalog/paimon.md [hunk 8]
```diff
@@ -30,31 +30,76 @@ under the License.
...
+    "s3.access_key"="ak",
+    "s3.secret_key"="sk"
 );
+
+```
+
+#### OSS
+
+```sql
+   CREATE CATALOG `paimon_oss` PROPERTIES (
```
Comment (kaijchen) [inline docs/zh-CN/docs/lakehouse/multi-catalog/paimon.md:77]:
```suggestion
CREATE CATALOG `paimon_oss` PROPERTIES (
```

File: docs/zh-CN/docs/lakehouse/multi-catalog/paimon.md [hunk 9]
```diff
@@ -30,31 +30,76 @@ under the License.
...
+* filesystem（默认），同时存储元数据和数据在filesystem。
+* hive metastore，它还将元数据存储在Hive metastore中。用户可以直接从Hive访问这些表。
 
-使用Paimon API访问元数据的方式，目前只支持Hive服务作为Paimon的Catalog。
+### 基于FileSystem创建Catalog
 
-- Hive Metastore 作为元数据服务
+#### HDFS
+```sql
+ CREATE CATALOG `paimon_hdfs` PROPERTIES (
```
Comment (kaijchen) [inline docs/zh-CN/docs/lakehouse/multi-catalog/paimon.md:48]:
```suggestion
CREATE CATALOG `paimon_hdfs` PROPERTIES (
```

**Review by kaijchen — COMMENTED** [review]
LGTM, please fix the indents

**Comment by zddr** [comment]
run buildall

**Review by kaijchen — APPROVED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**